sympy/geometry/entity.py

```diff
@@ -171,7 +171,7 @@ def _repr_svg_(self):
             # will fall back to the next representation
             return None
 
-        if any([not x.is_number or not x.is_finite for x in bounds]):
+        if any(not x.is_number or not x.is_finite for x in bounds):
```

Comment:
Might be clearer and faster as
```python
if not all(x.is_number and x.is_finite for x in bounds):
```
More importantly it might be more correct. What should happen here if `x.is_finite` gives `None`?

Reply:
I do not analyze the code to that extent... :-) But I do indeed see your point!